documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -6,21 +6,13 @@
 = Recovering a deleted cluster from persistent volumes
 
 [role="_abstract"]
-This procedure describes how to recover a deleted cluster from persistent volumes (PVs).
+This procedure describes how to recover a deleted cluster from persistent volumes (PVs) by recreating the original `PersistentVolumeClaim` (PVC) resources.
 
-In this situation, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
+If the Topic Operator is deployed, you can also recover `KafkaTopic` resources.
+This must be done before the cluster is started so that the corresponding topics are not deleted.
```

Comment:
This is not true anymore. The UTO will not delete the topics just because the CRs don't exist.